app/jobs/solidus_subscriptions/process_installment_job.rb

```diff
@@ -7,7 +7,17 @@ class ProcessInstallmentJob < ApplicationJob
     def perform(installment)
       Checkout.new(installment).process
     rescue StandardError => e
-      SolidusSubscriptions.configuration.processing_error_handler&.call(e)
+      error_handler = SolidusSubscriptions.configuration.processing_error_handler
+
+      if error_handler
+        proc_or_method = error_handler.is_a?(Proc) ? error_handler : error_handler.method(:call)
```

Comment:
Are we adding this arity check for compatibility with older versions? What's the purpose here?

I feel that this adds a bit of extra complexity (specially when comparing an arity of `-2`), and handling some logic which should not be the responsibility of this job.

Some users might also override the default class with one which requires both the installment and the error, in which case the arity would be `2`)

Reply:
I think you are right. I thought that an existing proc that only had one argument (like `proc { |e| raise e }`) wouldn't work but it seems that it handles not having the second argument without breaking. I'll remove the extra code, thanks!

Reply:
Oh, so we need to hope that people didn't use a lambda for their handler?

Reply:
Yes. But users who're upgrading should be able to deal with this right?

Reply:
Yes but if they didn't notice the change they'll end up with an exception in production, not ideal.